cmd/installer/cli/install.go

```diff
@@ -321,6 +319,15 @@ func runInstall(ctx context.Context, name string, flags InstallCmdFlags, metrics
 
 	// TODO (@salah): update installation status to reflect what's happening
 
+	logrus.Debugf("adding insecure registry")
+	registryIP, err := registry.GetRegistryClusterIP(flags.cidrCfg.ServiceCIDR)
+	if err != nil {
+		return fmt.Errorf("unable to get registry cluster IP: %w", err)
+	}
+	if err := airgap.AddInsecureRegistry(fmt.Sprintf("%s:5000", registryIP)); err != nil {
```

Comment:
curious why is this needed here since it's added as part of the registry chart installation

Reply:
I removed it from the registry chart installation. It seemed wrong there since its touching the host containerd conf files.